### ISSUE TITLE
Could I not update all the packages from bower when i only want update single composer package 

### ISSUE THREAD

**Issue by sevenT** [francoispluchino/composer-asset-plugin#94]
Could I not update all the packages from bower when i only want update single composer package ?

**Comment by francoispluchino** [comment]
Alas no, Composer retrieves a complete list of all packages. This is the native behavior of Composer.

**Comment by cebe** [comment]
`composer update myvendor/mypackage` ?

**Comment by francoispluchino** [comment]
Yes, in internal, Composer retrieves always all package definitions... even for a single package. In view of the previous discussions, I do not think it will change quickly.

**Comment by SDKiller** [comment]
> Composer retrieves always all package definitions... even for a single package.

Confirm, had the similar issue.

> I do not think it will change quickly

@francoispluchino 

May be offtopic, but related to https://github.com/francoispluchino/composer-asset-plugin/issues/93 and many `composer` issues not solved and `nice-to-have` proposals still not implemented - don't you think it's time to fork `composer` ? ;-)

**Comment by sevenT** [comment]
@cebe that's exact what i did to update my local package.
@francoispluchino Thanks for this awesome project.
It seems like a composer problem. The awful internet of CHN really drive me crazy.

**Comment by cebe** [comment]
@sevenT there is a local composer cache, when you use that you might need to download each package only once: https://toranproxy.com/

**Comment by sevenT** [comment]
Thanks @cebe  we've build up a local cache with satis, but only for php package. It still try to download extra asset packages. Maybe I should try to cache bower packages as well, I'll try this later when i got time and getting more familiar with how all these things work.  Many Thanks

**Comment by dynasource** [comment]
Im a really heavy use of "composer update myvendor/mypackage" but this isnt possible anymore, although it is still supported (see https://github.com/composer/composer/issues/450)

Is this issue solved by https://github.com/francoispluchino/composer-asset-plugin/issues/93?

**Comment by francoispluchino** [comment]
@dynasource This could be fixed by this issue.

**Comment by dynasource** [comment]
Excellent! Whats left to do to benefit from your patch?
thanks

**Comment by francoispluchino** [comment]
I must develop the new version 2.0 that used Nodejs. So for the moment this is not available.

**Comment by nsams** [comment]
Or you use https://github.com/koala-framework/composer-extra-assets which available now.

**Comment by francoispluchino** [comment]
@nsams It is also a solution.
